kedro_airflow_k8s/operators/node_pod.py

```diff
@@ -96,6 +98,9 @@ def __init__(  # pylint: disable=too-many-arguments
             is_delete_operator_pod=True,
             pod_template_file=self.minimal_pod_template,
             node_selector=node_selector_labels,
+            node_selectors=node_selector_labels,
```

Comment:
Is the repetition to line 100 necessary? It is assigned to the same k8s operator attribute in the end.

Reply:
fixed